src/app/src/pages/ForgotPassword.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import { Box, Button, Input, Text, VStack } from '@chakra-ui/react';
+import { useNavigate } from 'react-router-dom';
+
+import { API } from '../api';
+import { API_URLS } from '../constants';
+
+export default function ForgotPassword() {
+    const [email, setEmail] = useState('');
+    const [success, setSuccess] = useState(false);
+    const [errorDetail, setErrorDetail] = useState('');
+    const navigate = useNavigate();
+
+    const formatApiError = apiErrorData => {
+        var errorDetail = '';
+        Object.values(apiErrorData).forEach(val => (errorDetail += `${val}`));
```

Comment:
Should we use some sort of separator here? A newline or comma or something?

Reply:
I wondered too, but this is a one-field API (email), so I don't think we will run into that need in practice.

Reply:
I should add a comment at the least!

Reply:
Enh, it's easy enough to just join. I'm a believer now. 👍🏻